src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -52,13 +52,14 @@ export interface ShippingState {
 }
 
 export class ShippingRoute extends Component<ShippingProps, ShippingState> {
-  // TODO: Fill in with Relay data on load.
-  // See: https://artsyproduct.atlassian.net/browse/PURCHASE-376
   state = {
-    shippingOption: "SHIP" as OrderFulfillmentType,
+    shippingOption: ((this.props.order.requestedFulfillment &&
+      this.props.order.requestedFulfillment.__typename.toUpperCase()) ||
+      "SHIP") as OrderFulfillmentType,
```

Comment:
not really related to this PR, but it might make sense to align capitalization across the stack?

Reply:
Yeah I was wondering that too. I think the problem is we have two types (`Ship` and `Pickup`) as well as two enums with the same name (`SHIP` and `PICKUP`), but types and enums have different capitalization conventions. It's fairly minor, but if we had to do it over again, I'd say that the enums should be more verbose; for example, `SHIP_FULFILLMENT` or something. But that might just be the Objective-C developer in me talking 😉 /cc @ashkan18

Reply:
heheh im not sure here either. I think the recommendation was to use all caps for enums but i can be totally wrong :)
Here is where we create that enum -> https://github.com/artsy/exchange/blob/master/app/graphql/types/order_fulfillment_type_enum.rb#L2

Reply:
yeah, enums in GraphQL are all caps: https://graphql.org/learn/schema/#enumeration-types